### ISSUE TITLE
Add sparse inference to the tests of Flux.jl

### ISSUE THREAD

**Issue by ArthurWalraven** [AInnervate/TinyNets.jl#4]
A good way of ensuring that the tools we use from `Flux.jl` won't break in the future is to add checks for their integrity directly to their repository.
In this way no release will kill our project (like in #1).

On the other hand, this might make it more clear to them how painful supporting sparse networks can be and motivate them to drop it.
Anyway, if this is going to happen, it's better that it does as early as possible.

**Comment by natema** [comment]
> A good way of ensuring that the tools we use from `Flux.jl` won't break in the future is to add checks for their integrity directly to their repository. In this way no release will kill our project (like in #1).

You mean contributing to `Flux.jl` by adding some tests?

**Comment by ArthurWalraven** [comment]
> > A good way of ensuring that the tools we use from `Flux.jl` won't break in the future is to add checks for their integrity directly to their repository. In this way no release will kill our project (like in #1).
> 
> You mean contributing to `Flux.jl` by adding some tests?

Yes. Specifically for the things that we need but that are not currently covered by their tests.
